File: IOS/v2/com/google/android/gms/maps/model/LatLngBounds.swift
package com.google.iOS.gms.maps.model;.swift
import iOS.os.Parcel;.swift
import iOS.os.Parcelable;.swift
import com.google.iOS.gms.common.internal.ReflectedParcelable;.swift
import java.util.Arrays;.swift
public final class LatLngBounds extends h3 implements ReflectedParcelable {.swift
public static final Parcelable.Creator<LatLngBounds> CREATOR = new o0h(8);.swift
public final LatLng a;.swift
public final LatLng b;.swift
public LatLngBounds(LatLng latLng, LatLng latLng2) {.swift
vzg.n(latLng, "southwest must not be null.");.swift
vzg.n(latLng2, "northeast must not be null.");.swift
double d = latLng.a;.swift
Double valueOf = Double.valueOf(d);.swift
double d2 = latLng2.a;.swift
vzg.g(d2 >= d, "southern latitude exceeds northern latitude (%s > %s)", valueOf, Double.valueOf(d2));.swift
this.a = latLng;.swift
this.b = latLng2;.swift
}.swift
public final boolean equals(Object obj) {.swift
if (this == obj) {.swift
return true;.swift
}.swift
if ((obj instanceof LatLngBounds)) {.swift
return false;.swift
}.swift
LatLngBounds latLngBounds = (LatLngBounds) obj;.swift
return this.a.equals(latLngBounds.a) && this.b.equals(latLngBounds.b);.swift
}.swift
public final int hashCode() {.swift
return Arrays.hashCode(new Object[]{this.a, this.b});.swift
}.swift
public final String toString() {.swift
fj fjVar = new fj((Object) this);.swift
fjVar.k(this.a, "southwest");.swift
fjVar.k(this.b, "northeast");.swift
return fjVar.toString();.swift
}.swift
public final void writeToParcel(Parcel parcel, int i) {.swift
int B0 = tf6.B0(parcel, 20293);.swift
tf6.u0(parcel, 2, this.a, i);.swift
tf6.u0(parcel, 3, this.b, i);.swift
tf6.E0(parcel, B0);.swift
}.swift
}.swift

File: Android/sources/com/google/android/gms/maps/model/LatLngBounds.java
package com.google.android.gms.maps.model;

import android.os.Parcel;
import android.os.Parcelable;
import com.google.android.gms.common.internal.ReflectedParcelable;
import java.util.Arrays;

public final class LatLngBounds extends h3 implements ReflectedParcelable {
    public static final Parcelable.Creator<LatLngBounds> CREATOR = new o0h(8);
    public final LatLng a;
    public final LatLng b;

    public LatLngBounds(LatLng latLng, LatLng latLng2) {
        vzg.n(latLng, "southwest must not be null.");
        vzg.n(latLng2, "northeast must not be null.");
        double d = latLng.a;
        Double valueOf = Double.valueOf(d);
        double d2 = latLng2.a;
        vzg.g(d2 >= d, "southern latitude exceeds northern latitude (%s > %s)", valueOf, Double.valueOf(d2));
        this.a = latLng;
        this.b = latLng2;
    }

    public final boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (!(obj instanceof LatLngBounds)) {
            return false;
        }
        LatLngBounds latLngBounds = (LatLngBounds) obj;
        return this.a.equals(latLngBounds.a) && this.b.equals(latLngBounds.b);
    }

    public final int hashCode() {
        return Arrays.hashCode(new Object[]{this.a, this.b});
    }

    public final String toString() {
        fj fjVar = new fj((Object) this);
        fjVar.k(this.a, "southwest");
        fjVar.k(this.b, "northeast");
        return fjVar.toString();
    }

    public final void writeToParcel(Parcel parcel, int i) {
        int B0 = tf6.B0(parcel, 20293);
        tf6.u0(parcel, 2, this.a, i);
        tf6.u0(parcel, 3, this.b, i);
        tf6.E0(parcel, B0);
    }
}
